GVRf/IO/STL/src/org/gearvrf/stl/STLConstants.java

```diff
@@ -0,0 +1,26 @@
+/* Copyright 2015 Samsung Electronics Co., LTD
```

Comment:
remind me again what STL stands for?  as a c++ programmer (prior to working at Samsung), I see STL and I think standard template library.